server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -531,7 +531,7 @@ public void testTableConstraintsWithOrderBy() {
     @Test
     public void testDefaultColumns() {
         execute("select * from information_schema.columns order by table_schema, table_name");
-        assertEquals(858, response.rowCount());
+        assertEquals(863, response.rowCount());
```

Comment:
mistake? or how is this related?

Reply:
The settings are exposed as columns - (also just force pushed another change because the doc-tests were failing)

Reply:
ah by `sys.cluster.settings` right, sry.